src/abi/CoolerConsolidation.json

```diff
@@ -54,58 +54,6 @@
       ],
       "stateMutability": "view"
     },
-    {
```

Comment:
No need to remove these?

Reply:
@0xJem , these aren't in the latest abi i grabbed. that's why they're removed